### PR TITLE
Apply GCV to fix publications

### PR DESCRIPTION
**Pull request by carterkozak**
==COMMIT_MSG==
Apply GCV to fix publications
==COMMIT_MSG==

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -70,12 +72,6 @@ dependencyRecommendations {
     propertiesFile file: project.rootProject.file('versions.props')
```
Comment (ferozco) [inline build.gradle]: you'll want to remove this too, and add `com.palantir.baseline-versions.disable = true` to gradle properties

Comment (carterkozak) [reply build.gradle]: Ah, thanks!

**Comment by carterkozak** [comment]
Fixed, let's ship it.

**Comment by ferozco** [comment]
👍

**Pull request merged**